src/main/java/discord4j/discordjson/json/WebhookExecuteRequest.java

```diff
@@ -0,0 +1,28 @@
+package discord4j.discordjson.json;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import discord4j.discordjson.possible.Possible;
+import org.immutables.value.Value;
+
+import java.util.List;
+
+@Value.Immutable
+@JsonSerialize(as = ImmutableWebhookExecuteRequest.class)
+@JsonDeserialize(as = ImmutableWebhookExecuteRequest.class)
+public interface WebhookExecuteRequest {
+
+    static ImmutableWebhookExecuteRequest.Builder builder() {
+        return ImmutableWebhookExecuteRequest.builder();
+    }
+
+    Possible<String> content();
+    Possible<String> username();
+    @JsonProperty("avatar_url")
+    Possible<String> avatar_url();
```

Comment:
Shouldn't this be avatarUrl?

Reply:
Oof, you are right. I will get a pull request for that asap XD

Reply:
Fixed

Reply:
thanks